project-reports/2024/2024-annual-Hyperledger-Solang.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Hyperledger Solang
+parent: 2024
+grand_parent: Hyperledger TOC
+---
+
+# Project Health
+
+The 2024 forecast for Solang is showing growth despite the number of pull requests and reported issues going down, Here are the three reasons why:
+
+1. Stellar organization decided to fund Solang long term, to add Soroban as a compilation target.
+2. Work is ongoing in [Hyperledger Solang Playground](https://github.com/hyperledger-labs/solang-playground), a web IDE that will enable easy onboarding to the Solang ecosystem. Funding is provided by the Web3 Foundation.
+3. [Hyperledger Solang-Aqd](https://github.com/hyperledger/solang-aqd) (a smart contract CLI tool) was developed in the 2023 mentorship, by Tarek Naser, and it will be integrated into Solang Playground.
+
+# Maintainer Diversity
+- Another maintainer from a different organization has been added: Salaheldin Soliman. His main tasks are to add Soroban as a compilation target and on-boarding production projects to Solang.
+This makes Solang have 4 maintainers from 3 different organizations: Solana (Sean Young and Lucas Steuernagel), Parity (Cyril Leutwiler), and Stellar (Salaheldin Soliman).
+
+- Sean Young and Lucas Steuernagel have been redeployed to other projects within Solana, but they will continue contributing to Solang by reviewing pull requests and addressing issues.
+
+# Project Adoption
+
+The maintainer team has identified the issue with Solang's slow adoption in 2023: the lack of developer tooling and advocacy. Although a lot of work has gone into making a full-on compiler for Solidity, we need to provide easy ways for Solidity developers to start using Solang. The maintainer team will work on tutorials and easy-to-use tools to supplement Solang.
+
+
+# Goals
+
+## Performance Against Prior Goals
+Prior goals for Solang mainly involved making Solang more attractive to Solidity developers as stated in the 2023 [Q4 plans](https://github.com/hyperledger/toc/blob/gh-pages/project-reports/2023/2023-Q4-Hyperledger-Solang.md#current-plans)
+
+There have been many contributions to that cause, but they involved Solang itself and not its sidecar tools.
+
+
+## Next Year's Goals
+The main goal is to on-board more users and production projects to Hyperledger Solang. More work will be put into Solang Playground and Solang-Aqd to make it easier for Solidity developers to start using Solang, as well as tutorials to encourage production projects to use Solang.
```

Comment:
One of the suggestions from the TOC is to work on maintainer diversity as well as commit diversity.